src/ngraph/op/fused/rnn_cell.hpp

```diff
@@ -132,6 +132,7 @@ namespace ngraph
                     const std::vector<float>& activations_beta = {},
                     float clip = 0.f);
 
+                using RNNCellBase::visit_attributes;
```

Comment:
Why do you use using? It also will work without this.

Reply:
`RNNCell` operator have multiple inheritance and because of that `visit_attributes` will be ambiguous(`op::visit_attributes` and `RNNCellBase::visit_attributes`).